README.md

```diff
@@ -1,9 +1,5 @@
 # curriculum-databases-projects-template
 
-> This template should be used for database related projects at Microverse.
-> Generate your own repository, update this README and edit all files content while working on projects. You should not be adding any new files unless asked otherwise.
-
-
 ## Getting Started
 
 This repository includes files with plain SQL that can be used to recreate a database:
```

Comment:
- [ ] Good job adding customizing this section 👍 , however you misplaced it , Kindly put it in the right place as represented in the README template , make sure you don't have it at the top of the table of contents 👍

Reply:
Hello @iambenkis ! Thanks for the feedback, could you guide me regarding this suggestion? Where it should go since in fact, this is how it is found in the readme of the template, I hope you can help me with that question, regards!